### PR TITLE
add tests for SMA

### PR DESCRIPTION
**Pull request by LeeDongGeon1996**
## Description

Add tests for SMA indicator.

 * Focused on keeping the style of original C# code.
 * Test Framework: `unittest`
 * Test Runner: `pytest`

Implements #394 

## Checklist

- [x] My code follows the existing style, code structure, and naming taxonomy
- [x] I have performed a self-review of my own code and included any verifying manual calculations
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, including `INDICATORS.md`, `README.md`, `info.xml`, etc
- [ ] I have made corresponding changes to the `wraps` interoperability files
- [x] My changes generate no new warnings and running code analysis does not produce any issues
- [x] I have added or updated unit tests that prove my fix is effective or that my feature works, and achieves sufficient code coverage
- [ ] I have added or run the performance tests that depict optimal execution times
- [x] New and existing unit tests pass locally and in the build (below) with my changes

## Acknowledgements

- [x] I have read and understand [the Apache 2.0 license](https://opensource.org/licenses/Apache-2.0)
- [x] I agree to the [Developer Certificate of Origin (DCO) 1.1](https://developercertificate.org)

### CODE REVIEW

**Comment by LeeDongGeon1996** [comment]
You can run the test using:
```python
python -m unittest -v SkenderStockIndicators.tests.test_sma
```

**Comment by DaveSkender** [comment]
I'll take a deeper look at this over the weekend, some initial observations/questions:

1. Can you reference the original spreadsheet, instead of making a copy?  Mostly just want to avoid having to synchronize these if one changes.
2. Can we run all tests and produce publishable test results?  I'm thinking of how we can add this to the build process (next steps).

**Comment by LeeDongGeon1996** [comment]
1. There is no problem to reference the original, but as well as dll, the spreadsheet has to be included in our production package at the end so that every test can fetch data from the spreadsheet.
2. I may need to do some more commit in order to run all tests with a single command. Then we can publish the test results from single command. Maybe this is what you want to do:  [Build Python apps](https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/python?view=azure-devops)

**Comment by DaveSkender** [comment]
> * There is no problem to reference the original, but as well as dll, the spreadsheet has to be included in our production package at the end so that every test can fetch data from the spreadsheet.

I would not expect tests and test data _in_ the production package at all, but rather something used _on_ the production package.

**Comment by LeeDongGeon1996** [comment]
[[`4561f22`](https://github.com/DaveSkender/Stock.Indicators/pull/421/commits/4561f22d0cf134e81d69680343eb8382d0521447)] I tried some tests about it on TestPyPI and figured out that we can exclude tests from production package just by moving it to out of `SkenderStockIndicator` directory.

FYI:
But, some of well-known Python packages, they distribute their production package with their test code. 
You can check which Python packages are distributed with their test code;  `python -m site` shows where Python packages are installed.

**Comment by DaveSkender** [comment]
Request a review when you're ready to proceed on this one.

**Comment by LeeDongGeon1996** [comment]
Ready to be reviewed.

**Comment by DaveSkender** [comment]
Can you start a README.md file with instructions on how to run unit tests?  I'm only getting errors, and I'm fairly sure it's because I'm not doing something right.

```
$ python -m unittest -v tests.test_sma
test_sma (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: test_sma (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_sma
Traceback (most recent call last):
  File "C:\Program Files\Python38\lib\unittest\loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
ModuleNotFoundError: No module named 'tests.test_sma'
```

**Comment by LeeDongGeon1996** [comment]
No problem, I'll do that by this Wednesday

**Comment by LeeDongGeon1996** [comment]
[[`740f90b`](https://github.com/DaveSkender/Stock.Indicators/pull/421/commits/740f90b67e47bd90ccd5f54df6f592ba9fd38aa0)] We are going to use `pytest`, which is easy to use and fully compatible with `unittest`,  as a test runner.

File: wraps/python/README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
 # Stock Indicators for Python
 
-[Stock Indicators for Python]() is a Python wrapper for [Stock Indicators](https://github.com/DaveSkender/Stock.Indicators), written in C#.
\ No newline at end of file
+[Stock Indicators for Python]() is a Python wrapper for [Stock Indicators](https://github.com/DaveSkender/Stock.Indicators), written in C#.
+
+## Testing
+`pytest` is required for testing.
+```python
+pip install pytest
```
Comment (DaveSkender) [inline wraps/python/README.md]: What's the difference between `pip` and `pip3` and does it make a difference here?

To get it to work, after installing Python 3.8 and `pytest`, I also had to:
```
pip3 install openpyxl
```

**Review by DaveSkender — CHANGES_REQUESTED** [review]

File: wraps/python/README.md (hunk 1 above)
Comment (LeeDongGeon1996) [reply wraps/python/README.md]:
```suggestion
pip install pytest openpyxl
```

Do you also have Python2 installed? If so, `pip` works for Python2, `pip3`works for Python3. Usually `pip3` can be replaced with `pip` if you have only Python3.

Anyway, we are using Python3, you should use `pip3` as shown above.

**Review by DaveSkender — APPROVED** [review]

**Pull request merged**